frontend/src/helpers/prefetch.ts

```diff
@@ -4,11 +4,10 @@ import { queryFooter, queryHeader } from "~/types";
 
 export const prefetchSharedQueries = async (
   client: ApolloClient<any>,
-  language: string,
+  _language: string,
```

Comment:
language is not needed anymore since we fetch all menu items now. but I don't want to remove it since it might be better in future and don't want to edit 100 files again 😓